cns/restserver/restserver_test.go

```diff
@@ -689,3 +689,23 @@ func TestGetInterfaceForNetworkContainer(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
```

Comment:
Add a comment to describe the function

Reply:
Can you respond to the review comment rather than marking it resolved directly if you are not going to act on it ?

Reply:
Apparently, none of my responses got saved. I think Test name is self explanatory , hence no comment needed.